src/app/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './assets/css/custom.css';
```

Comment:
arrange imports by name. Leave extra lines only between third party imports and our imports.